isis/src/base/apps/csminit/csminit.cpp

```diff
@@ -417,6 +423,29 @@ namespace Isis {
     blobLabel += PvlKeyword("PluginName", pluginName);
     cube->write(csmStateBlob);
 
+    // Try to attach rotations
+    if (ui.WasEntered("ISD")) {
+      QString isdFilePath = ui.GetFileName("ISD");
+      std::ifstream ifs(isdFilePath.toStdString());
+      json jf = json::parse(ifs);
+
+      Spice spice(*cube->label(), jf);
+
+      Table bodyTable = spice.bodyRotation()->Cache("BodyRotation");
```

Comment:
Instead of calling the Spice constructor and all of the extra code that entails, can we call Table::LoadCache(isd)?

Reply:
The frame code for constructing the SpiceRotation object is the first frame in body_rotation/time_dependent_frames. We could also add a new SpiceRotation constructor that just takes a JSON object to encapsulate all of the json parsing in SpiceRotation